EIPS/eip-7524.md

```diff
@@ -0,0 +1,164 @@
+---
+eip: 7524
```

Comment:
```suggestion
eip: 7537
```
Assigning next sequential EIP/ERC number.  ERC7524 is already in use in #7775
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.